2sem/lab1/lab1.cpp

```diff
@@ -0,0 +1,28 @@
+﻿#include <iostream>
+#include <stack.h>
+using namespace std;
+
+int main()
+{
+    setlocale(LC_ALL, "");
+	int numberForEnter=4;
+	Stack<int> stack;
+
+	while (numberForEnter != 0)
+	{
+		cout << "Введите число: ";
+		cin >> numberForEnter;
+		if (numberForEnter != 0)
+		{
+			stack.push(numberForEnter);
+		}
+	}
+
+	while (stack.count()) {
```

Comment:
Надо привести код к требованиям стайл гайда
```suggestion
	while (stack.count()) 
	{
```

Reply:
Исправлено